izanami-frontend/src/App.tsx

```diff
@@ -628,7 +629,7 @@ export class App extends Component {
   };
   constructor(props: any) {
     super(props);
-
+    window.localStorage.clear();
```

Comment:
This clears localStorage each time Izanami is opened, which is not what we want here, since the point is to keep last search in memory across visits.
Another side effect of this is that search history will be lost if you open a new tab in the same Izanami instance.